sources/packages/web/src/services/http/ApplicationChangeRequestApi.ts

```diff
@@ -0,0 +1,18 @@
+import { ApplicationChangeRequestAPIInDTO } from "@/services/http/dto/ApplicationChangeRequest.dto";
+import HttpBaseClient from "./common/HttpBaseClient";
```

Comment:
Please use the following way to import the `HttpBaseClient`

`import HttpBaseClient from "@/services/http/common/HttpBaseClient";`